### PR TITLE
Parent graphs

### PR DESCRIPTION
**Pull request by AlphaPranav9102**
Changed so graphs are passed in a parent container and self-generate without the need for individual html containers having to be generated for each graph. reduces boilerplate and setup effort.

### CODE REVIEW

File: src/components/BarGraph.js [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
         this.modal = modal
 
-        this.container = document.getElementById(id)
-        this.container.setAttribute("type", "button")
-        this.container.setAttribute("data-modal-toggle", this.uuid)
+        this.companionDiv = document.createElement("div")
+        for (const style of ["p-4", "border-2", "border-gray-200", "rounded-lg"]){
+            this.companionDiv.classList.add(style)
+        }
+        this.companionDiv.style.width = "400px"
```
Comment (sswadkar) [inline src/components/BarGraph.js:13]: why px and not viewport width or rem or something

File: src/components/LineGraph.js [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
         this.modal = modal
 
-        this.container = document.getElementById(id)
-        this.container.setAttribute("type", "button")
-        this.container.setAttribute("data-modal-toggle", this.uuid)
+        this.companionDiv = document.createElement("div")
+        for (const style of ["p-4", "border-2", "border-gray-200", "rounded-lg"]){
+            this.companionDiv.classList.add(style)
+        }
+        this.companionDiv.style.width = "400px"
```
Comment (sswadkar) [inline src/components/LineGraph.js:13]: read comment above

File: src/components/BarGraph.js (hunk 1 above)
Comment (AlphaPranav9102) [reply src/components/BarGraph.js:13]: I'm pretty sure em, rem and the other relatives are just a certain multiplier or pixels so like 16 px is 1rem. so it's just a smaller number.

VW is relative to screen but I want these cards to stay the same width no matter what so flexbox can resize and make grid patterns . if I set card to 10 VW then it's like 10% so in mobile it's mega small

File: src/components/LineGraph.js (hunk 2 above)
Comment (AlphaPranav9102) [reply src/components/LineGraph.js:11]: added

**Pull request merged**